Split log forwarding into separate topics

mburke5678 opened the pull request.

sichvoge commented on _topic_map.yml:
```diff
@@ -1190,9 +1190,12 @@ Topics:
+- Name: Forwarding logs using the Log Forwarding API
   File: cluster-logging-external
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
+- Name: Forwarding logs using a ConfigMap
```
Seeing it up here makes me not like it anymore :D Maybe we should keep "Forwarding logs to third party systems" for now and describe the ConfigMap approach here, as it is whats fully supported today, and then leave "Forwarding logs using the Log Forwarding API" as a separate chapter as is. 

Also, the "to third party systems" should come before the log forwarding API.

WDYT?

mburke5678 replied:
```
Forwarding logs to Third-Party System
Forwarding logs using the Log Forwarding API
```
If the first topic tells folks how to send logs to external systems, what does the Log Forward API do? <smiley-face>

mburke5678 replied:
_ the API is just a CRD that the CLO takes and generates the corresponding Fluentd config._
I did not know this. (Is this true when the API goes GA? We use the Fluentd config map to send logs to Kafka, syslog, etc?)

mburke5678 replied:
_When you use the Log Forwarding API, the Cluster Logging Operator creates or modifies a Fluentd ConfigMap to send logs using the Fluentd *forward* protocol. You are responsible to configure the protocol on the external log aggregator. _

sichvoge commented on logging/cluster-logging-external-configmap.adoc:
```diff
@@ -0,0 +1,26 @@
+
+[NOTE]
+====
+Forwarding logs using a ConfigMap will be deprecated when the xref:../logging/cluster-logging-external.adoc#cluster-logging-external[Log Forwarding API] reaches General Availability (GA) status, and will be removed in a future release.
```
s/when/once?

mburke5678 replied:
The IBM Style Guide recommends using _once_ to mean _one time_.

mburke5678 commented:
@sichvoge Any final thoughts on this? I would like to get this PR merge ahead of other changes to these files. 

I changed the wording around deprecating the legacy configmap methods because of the additional text we added that the LF API uses a fluentd configmap. I thought it odd that in one place we say configmaps to forward logs will be deprecated when the LF API is GA, but the LF API uses configmaps.

_Manually configuring a ConfigMap to forward logs will be deprecated when the Log Forwarding API reaches General Availability (GA) status and will be removed in a future release._
